CHANGELOG.md

```diff
@@ -8,11 +8,84 @@
 
 ## Unreleased
```

Comment:
I think this file was accidentally included in this PR, could you remove this change from the PR? thank you!

Reply:
I think @Lms24 accidentally added this file to this PR.

Reply:
Good catch, thx! This probably happened b/c of the target branch change